docs/src/developer/documenting.rst

```diff
@@ -1,28 +1,16 @@
-.. contents:: Developer's Guide 102: Documenting
+***********
+Documenting
+***********
+The documentation is built using Sphinx_ with the `Read The Docs`_ theme.
 
-********
-Document
-********
+To generate the *html* and *man* pages of the documentation, run:
 
-We are using `Read The Docs`_ theme for Sphinx_.
-
-Run tox command::
+.. code-block:: sh
 
    tox -e docs
 
-to build *html* and *man* pages for documentation
-
-Also by executing::
-
-   tox -e serve-docs
-
-the page under ``/docs/build/html`` is hosted by *localhost*.
-
-Use also command::
-
-   tox -e doc8
-
-to ensure that documentation standards are being followed.
+When writing, you can also run ``$ tox -e serve-docs`` to host the content of
+``/docs/build/html`` on http://localhost:8000.
```

Comment:
There should be more details in this section. The addition of new modules in docs/src/code for instance. How things are structured. What kind of examples we would be looking for.

Reply:
I agree with this suggestion. I was planning to add this content in https://trello.com/c/bqcySHQr in order to not have a PR that's too big.

In the meantime, for this PR, I agree that's the doc looks too stripped down so I'm going to add the module documentation snippet you mentioned and give a brief overview.

Reply:
Oh I didn't notice this trello card. Sounds good! I'll mark this thread as resolved.